src/OSPSuite.R/Domain/SimulationBatch.cs

```diff
@@ -31,6 +31,9 @@ public class SimulationBatchOptions
 
    public class SimulationBatchRunValues
    {
+      //Id to recognize it when running concurrently
+      public string Id { get; set; }
```

Comment:
Who is setting this? 
How is this going to be used? I don't understand

Reply:
Is this set maybe by the caller in R? What happened if this is not set?

Reply:
This is an Id that the caller from R should fill. This is what I explained to you after discussing with Pavel. The list of results is created in a way that you can infer the order but still could be a bit tricky and the best way is you assign a set of ids that you can later compare, so a given result will match the id of a given SimulationBatchRunValues.

Reply:
Now the id is retrieved when the options are added